### DISCUSSION TITLE
how to code to import skin on osu Lazer Linux

### DISCUSSION THREAD

**Discussion by cool271** [ppy/osu#28742] [Q&A] [answered]
I just found out that when I can choose terminal in folder and it says '/mnt/chromeos/MyFiles/Downloads/Vaxei 2023 (1)$ ' in terminal Linux. anyone know what to do next? also i'm on chromebook so I can't do the normal way

**Comment by cool271** [comment]
can someone that knows coding on Linux help me?

**Comment by Stoppedpuma** [comment, marked as answer]
You already opened this discussion at #28722, please don't make knowingly duplicate discussions. Use the import option here:

![image](https://github.com/ppy/osu/assets/58333920/b9563c9b-a83d-48a1-936e-7dbcc2540074)
